src/divider.jsx

```diff
@@ -4,16 +4,22 @@ import {mergeStyles, prepareStyles} from './utils/styles';
 
 const propTypes = {
   /**
-   * CSS class that will be added to the divider's root element
+   * The css class name of the root element.
    */
   className: React.PropTypes.string,
 
   /**
-   * If true, the divider will be indented 72px
+   * If true, the `Divider` will be indented `72px`.
    */
   inset: React.PropTypes.bool,
 
   /**
+   * The material-ui theme applied to this component.
+   * @ignore
```

Comment:
The reason behind this PR is from this comment: https://github.com/callemall/material-ui/pull/3033#issuecomment-174619050

Also just change code formatting to follow that Subheader component in general.